### PR TITLE
Feat: Add Sentry

### PR DESCRIPTION
**Pull request by mul53**
- Add sentry to fuseswap

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -40,6 +43,16 @@ window.addEventListener('error', error => {
   })
 })
 
+const SENTRY_DSN: string | undefined = process.env.REACT_APP_SENTRY_DSN
```
Comment (leonprou) [inline src/index.tsx:46]: Could you put the needed values for Sentry to env.production?

**Review by leonprou — CHANGES_REQUESTED** [review]
Looks good. see my comment

File: src/index.tsx (hunk 1 above)
Comment (leonprou) [reply src/index.tsx:46]: they are not secret so I think it's the easiest way to go

Comment (mul53) [reply src/index.tsx:46]: Added the value

**Review by leonprou — APPROVED** [review]
👍

**Pull request merged**